src/network/clientpackethandler.cpp

```diff
@@ -1327,7 +1327,8 @@ void Client::handleCommand_HudSetStars(NetworkPacket *pkt)
 	StarParams stars;
 
 	*pkt >> stars.visible >> stars.count
-		>> stars.starcolor >> stars.scale;
+		>> stars.starcolor >> stars.scale
+		>> stars.visible_at_day;
```

Comment:
This will probably read `0` from the memory, resulting in corrupt values. Please check against the packet length prior reading this parameter.